app/controllers/application_controller.rb

```diff
@@ -1,7 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
-  helper_method :cart, :current_user, :current_default?
+  helper_method :cart, :current_user, :current_default?, :current_merchant_employee?, :current_merchant_admin?, :current_admin?
```

Comment:
nice helper methods, i think i need to do this for a class method im using, but is there a limit to how many helper methods we can use at application level or page level?